### PR TITLE
Adding unit 7 lesson 4 & 5

### PR DESCRIPTION
**Pull request by rosiecakes**
Hope you don't mind my small PR's

Added test, main, readme, solution

### CODE REVIEW

File: unit-7-functions-intro/lesson-4-return-even-or-odd/solutions/solution.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+def even_or_odd(num):
+    if num % 2 == 0:
+        return 'even'
+    elif num % 2 !=0:
+        return 'odd'
+    else:
+        return 'number please!'
```
Comment (santiagobasulto) [inline unit-7-functions-intro/lesson-4-return-even-or-odd/solutions/solution.py]: This case is not covered in the description or tests.

Comment (rosiecakes) [inline unit-7-functions-intro/lesson-4-return-even-or-odd/solutions/solution.py]: Removed

**Comment by rosiecakes** [comment]
Forgot to close PR before adding next lesson. Please review both lessons 4 & 5 @santiagobasulto

**Comment by rosiecakes** [comment]
Please review small assignment 6 also

**Comment by santiagobasulto** [comment]
+1

**Pull request merged**